internal/transport/transport_test.go

```diff
@@ -2136,6 +2137,69 @@ func (s) TestHeadersHTTPStatusGRPCStatus(t *testing.T) {
 	}
 }
 
+func (s) TestWriteHeaderConnectionError(t *testing.T) {
+	server, client, cancel := setUp(t, 0, notifyCall)
+	defer cancel()
+	defer server.stop()
+
+	waitWhileTrue(t, func() (bool, error) {
+		server.mu.Lock()
+		defer server.mu.Unlock()
+
+		if len(server.conns) == 0 {
+			return true, fmt.Errorf("timed-out while waiting for connection to be created on the server")
+		}
+		return false, nil
+	})
+
+	if len(server.conns) != 1 {
+		t.Fatal("Server must have an active connection for the client.")
+	}
+
+	// Get the server transfort for the connecton to the client
```

Comment:
Nit: s/transfort/tansport

Also, please terminate the comment with a period.

Reply:
Done.